examples/typescript/transfer_secondary_store.ts

```diff
@@ -0,0 +1,266 @@
+/* eslint-disable no-console */
+/* eslint-disable max-len */
+
+import {
+  Account,
+  AccountAddress,
+  AnyNumber,
+  Aptos,
+  AptosConfig,
+  ClientConfig,
+  InputViewFunctionData,
+  Network,
+  NetworkToNetworkName,
+} from "@aptos-labs/ts-sdk";
+import { compilePackage, getPackageBytesToPublish } from "./utils";
+
+/**
+ * This example demonstrate how one can compile, deploy, and transfer fungible assets (FA) between primary and secondary stores
```

Comment:
if this examples handles both primary and secondary, rename the file to something like `transfer_between_fungible_stores`